tests/Nelmio/Alice/Fixtures/LoaderTest.php

```diff
@@ -94,11 +88,12 @@ public function testGetBadReference()
 
     public function testLoadUnparsableFile()
     {
-        try {
-            $res = $this->createLoader()->load($file = __DIR__.'/../support/fixtures/not-parsable');
-        } catch (\UnexpectedValueException $e) {
-            $this->assertEquals("{$file} cannot be parsed - no parser exists that can handle it.", $e->getMessage());
-        }
+        $file = __DIR__.'/../support/fixtures/not-parsable';
+        $this->setExpectedException(
```

Comment:
Actually, I see that we're using annotations elsewhere - is there any reason you're choosing not to here?